### PR TITLE
Create a python module that records one line of location data to a csv file

### PR DESCRIPTION
**Pull request by ccason1**

### CODE REVIEW

**Comment by ccason1** [comment]
@ivogeorg Let me know if you need me to add or change anything.

**Comment by ivogeorg** [comment]
@ccason1 Great!

Two questions:
1. What is actually returned? WGS18? Could you pls include a sample return value in the script as a comment?
2. What is the overhead of running a subprocess and opening a file to write its output every time we need the current location? Is running a command-line process the only way to get this data?

**Comment by ccason1** [comment]
@ivogeorg I overhauled the whole thing so now it's using the actual Python libraries rather than command line arguments. The output is customizable now. I also commented an example output at the top of the code.

File: gps/record_flight_data.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        
+        self.data_output_options = list(data_output_options)
+        self.session = gps(mode=WATCH_ENABLE)
+        self.output_file = open(filename, 'a')
+        self.writer = csv.writer(self.output_file)
+        
+        # write the header to the file
+        self.writer.writerow(self.data_output_options)
+
+    def get_report(self):
```
Comment (ivogeorg) [inline gps/record_flight_data.py:41]: @ccason1 The name is too vague. Something closer to `write_current_packet` would be better. Think from the perspective of the library user. They don't know the internals and `write_date` could mean so many things.

File: gps/record_flight_data.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            report = self.session.next()
+            if report['class'] == 'DEVICE':
+                self.session.close()
+                self.session = gps(mode=gps)
+        except StopIteration:
+            return
+        
+        return report
+    
+    def write_data(self, timeout=1.5):
```
Comment (ivogeorg) [inline gps/record_flight_data.py:55]: @ccason1 The name is too vague. Something closer to `write_current_packet` would be better. Think from the perspective of the library user. They don't know the internals and `write_data` could mean so many things.

**Pull request merged**